### ISSUE TITLE
Trouble with Instant Clone

### ISSUE THREAD

**Issue by MajewskiPiotr** [vmware/vsphere-automation-sdk-rest#39]
Hello, 
I have a problem with Instant clone via REST API.
My request looks like this and I sent it to {**url**}/rest/vcenter/vm?action=instant-clone
{
    "spec": {
        "bios_uuid": "test4",
        "guest_OS": "WINDOWS_7_SERVER_64",
        "name": "Klon",
        "source": "vm-40316"
    }
}

in response i have :(

{
    "type": "com.vmware.vapi.std.errors.unexpected_input",
    "value": {
        "messages": [
            {
                "args": [
                    "[action]",
                    "operation-input"
                ],
                "default_message": "Found unexpected fields [action] in structure 'operation-input'.",
                "id": "vapi.data.structure.field.unexpected"
            }
        ]
    }
}

What am I doing wrong?

**Comment by jrgarcia** [comment]
So are you passing
```json
{
  "spec": {
    "bios_uuid": "test4",
      "guest_OS": "WINDOWS_7_SERVER_64",
      "name": "Klon",
      "source": "vm-40316"
  }
}
```
as a body set to `Content-Type: application/json` to `https://{vCenter URL}/rest/vcenter/vm?action=instant-clone`?

**Comment by MajewskiPiotr** [comment]
Exacly :)

**Comment by jrgarcia** [comment]
I tested this and got the same error. Just to be clear, this repository is mostly just samples for Postman. I will look into this and keep you updated with what I find though.

**Comment by MajewskiPiotr** [comment]
hi, maybe you managed to deal with the problem?

**Comment by jrgarcia** [comment]
It appears that this feature is not actually available yet from the REST API. It should be available in a future release though. I apologize for the inconvenience of that.

**Comment by MajewskiPiotr** [comment]
And when is the next version planned to be released?

**Comment by kmruddy** [comment]
No date has been announced yet.

**Comment by MajewskiPiotr** [comment]
Hi, I would like to remind myself in the subject :) I know that you have taught me that you do not know the date but the problem is urgent so I am looking for help from every side :)

**Comment by jrgarcia** [comment]
Looks like it's available in 7.0.
